core/include/scipp/core/view_index.h

```diff
@@ -56,12 +62,14 @@ class SCIPP_CORE_EXPORT ViewIndex {
     return m_fullIndex != other.m_fullIndex;
   }
 
-  constexpr bool has_stride_zero() const noexcept { return m_dims > m_subdims; }
+  [[nodiscard]] constexpr bool has_stride_zero() const noexcept {
+    return m_dims > m_subdims;
+  }
```

Comment:
`ViewIndex` used to be used by `transform`, but was replaced by `MultiIndex`. I think this method is unused now and can be removed.

Reply:
Yes, CLion reports it as unused. gonna remove it